ctfpad/views/files.py

```diff
@@ -51,3 +53,10 @@ def get(self, request, *args, **kwargs):
         form = self.form_class(initial=self.initial)
         return render(request, self.template_name, {'form': form})
 
+
+class ChallengeFileDownloadView(LoginRequiredMixin, View):
```

Comment:
There not really a need for CBV here if all you do is a GET which is a staticmethod. Maybe change here to FBV

Reply:
I am not really sure what you are talking about here, please elaborate ^^'